install conda envs on $PROJECT (#42)
C2SM/icon-vis: README.md

	<summary> <b><u> Instructions </u></b> </summary>

1. Look up most recent Miniconda version for Linux 64-bit on the [Miniconda documentation pages](https://docs.conda.io/en/latest/miniconda.html)
2. Install miniconda on your `$HOME` directory (default location);
        
       wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh

       bash Miniconda3-latest-Linux-x86_64.sh

> 3. Install all environments on your `$PROJECT` directory (Piz Daint) or on your `$SCRATCH` (Tsa), otherwise you risk filling up your $HOME directory.

Reviewer: ```suggestion
3. Install all environments on your `$PROJECT` directory (Piz Daint) or on your `$SCRATCH` (Tsa), otherwise you risk filling up your $HOME directory. See below for instructions.
```

</details>

### Create conda environment

> Create a conda environment _psyplot_ on your $PROJECT (Piz Daint) or $SCRATCH (Tsa) directory with python[version>=3.7,<3.10] (psy-view requirement) and install requirements:

Reviewer: ```suggestion
In the following instructions replace $PROJECT by $SCRATCH if using Tsa (instead of Piz Daint).

Create a conda environment _psyplot_ with python[version>=3.7,<3.10] (psy-view requirement) and install requirements:
```

    conda env create --prefix $PROJECT/envs/psyplot -f env/environment.yml

> To be able to activate your conda environment by simply using `conda activate psyplot` instead of the full path, add the following to your `.bashrc` (replace `$PROJECT` by `$SCRATCH` on Tsa):

Reviewer: ```suggestion
To be able to activate your conda environment by simply using `conda activate psyplot` instead of the full path, add the following to your `.bashrc`:
```
